data/json/monster_special_attacks/monster_attacks.json

```diff
@@ -314,5 +314,24 @@
     "miss_msg_npc": "%s tries to grab <npcname>, but they dodge!",
     "no_dmg_msg_u": "%s grasps at you with silk-covered legs, engulfing you in sticky webs!",
     "no_dmg_msg_npc": "%s grabs <npcname> and covers them in sticky webs!"
+  },
+  {
+    "type": "monster_attack",
+    "attack_type": "melee",
+    "id": "leg_sweep",
+    "cooldown": 10,
+    "move_cost": 100,
+    "attack_chance": 50,
+    "hitsize_min": 4,
+    "attack_upper": false,
+    "required_effects_any": [ "grabbing" ],
+    "damage_max_instance": [ { "damage_type": "bash", "amount": 10 } ],
```

Comment:
You'll either want higher damage or (and I'd argue for this) make the effect independent from the damage using `effects_require_dmg: false` to inconvenience anybody other than early characters.